internal/tjson/schema.go

```diff
@@ -57,6 +57,7 @@ const (
 	Byte     SchemaFormat = "byte"
 	UUID     SchemaFormat = "uuid"
 	DateTime SchemaFormat = "date-time"
+	Regex    SchemaFormat = "regex"
```

Comment:
Hmm, there is no "regex" format in Tigris' Schema Specification. Could you explain why it is here and how we are going to use it? I expect, if we try to create such a schema, Tigris will return an error. Have you tried it?

Reply:
This may be resulted from my still little knowledge about Tigris :smile:. The tests are passing as far as I understand they need to create some schema. Should I remove that line or just leave it with empty quotes?

Reply:
I don't understand why you need this line at all :) We store regex as an object. Where is this format from, and how do you use it? If you don't use it at all, let's not invent it.

Reply:
You're right